man/subscription-manager.8

```diff
@@ -616,7 +617,8 @@ option
 .B --list
 Lists all overrides from repositories specified with the
 .B --repo
-option
+option. Source of the listed overrides is a union of the override specified at the system itself and all the overrides set in the environment(s) system belongs in.
```

Comment:
minor correction: there is a missing `the` between the words `environment(s)` and `system`